### ISSUE TITLE
Rewrite OASIS+ comparison table copy

### ISSUE THREAD

**Issue by rosamundtgov** [GSA/oasis-plus#198]

**Comment by vickimcfadden** [comment]
@rosamundtgov confirmed with Esther & John this is okay to hold on until post-MVP.

**Comment by klohman** [comment]
@ekidenda @PowderD 

We've made some proposed edits to the How does OASIS+ compare to legacy OASIS table copy in this [draft document ](https://docs.google.com/document/d/1XM9P8iXhEhaOnuIjfQeB7t6-IX6mmi8NojAnmBnwgrc/edit)

Please review and add comments. I'm sure there are some oversimplifications, but the general format should be easier for users to scan and understand quickly.

**Comment by ekidenda** [comment]
@klohman  I've reviewed [How OASIS+ Compares - Website Page Content ] (https://docs.google.com/document/d/1TOzw0xzxMmnvWgBnEZpbMSCGpMPDSnVX1vMk0q8liOE/edit ) and believe the content is the same as the other document. I'd and accepted most of the suggested edits and would like to pass it by ConOps SME to ensure we did not leave important content out while simplifying - I will let them know the goal is to make it plain language based on UAT feedback.

**Comment by klohman** [comment]
@ekidenda @PowderD 

This page update is ready for your review [here](https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/preview/gsa/oasis-plus/klohman-198/oasis-plus/about/oasis-plus-comparison/)

I did make a tweak from the copy doc and added all of the similarities to this icon list. If you want to tweak the wording here let me know, but I think it works better than adding another table for the similarities. 

<img width="684" alt="Screenshot 2024-04-10 at 9 23 28 AM" src="https://github.com/GSA/oasis-plus/assets/18366192/e6258667-fcaf-4745-87de-a4babdd9f8f0">

**Comment by klohman** [comment]
@ekidenda Suggested edits made - [see updates here](https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/preview/gsa/oasis-plus/klohman-198/oasis-plus/about/oasis-plus-comparison/)

**Comment by PowderD** [comment]
Do we need the portion "similar to OASIS"? Isn't that redundant as the
header already reads "Similarities between legacy OASIS and OASIS+:"?

OASIS+ is on track to be designated as a Best-in-Class (BIC) contract  and
will help agencies meet their spend under management goals.

BTW, I totally defer to Esther for all grammar, just found that portion to
stick out to me.  So if Esther wants to keep it, let's move forward with
it!

*Jon DeWolfe  |  *
Program Manager Buyer Engagement
Services Marketplace
Federal Acquisition Service, U.S. General Services Administration
|(: 208.830.8517 | *: ***@***.***

**Comment by ekidenda** [comment]
Good catch @PowderD, you are right that it is redundant. Let's go with Jon's shorter version for sure @klohman

**Comment by klohman** [comment]
@ekidenda @PowderD I'm not sure I understand the redundancy. Can you place any additional edits in the [copy doc here](https://docs.google.com/document/d/1TOzw0xzxMmnvWgBnEZpbMSCGpMPDSnVX1vMk0q8liOE/edit) and I will update the page to match?

**Comment by klohman** [comment]
@ekidenda Made the changes from our discussion - preview is ready [here](https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/preview/gsa/oasis-plus/klohman-198/oasis-plus/about/oasis-plus-comparison/) for your review

**Comment by ekidenda** [comment]
Reviewed and this page looks good. Moving to done.
